docs/knowledge_base/dependency_file_exclusion.md

```diff
@@ -0,0 +1,176 @@
+# Dependency File Exclusion
+
+Analyzing dependencies with Phylum is possible with one of three primary solutions:
+
+* The [Phylum CLI][cli]
+* The [`phylum-ci` tool][phylum_pypi], usually in the form of [integrations]
+* The [Phylum GitHub App][gh_app]
```

Comment:
Pretty sure you can also submit lockfiles for analysis via the UI.

Reply:
Ah, yes, you are correct. The page will be updated to add that option.